docs/Catalog/Packaging.md

```diff
@@ -0,0 +1,189 @@
+`aws.omics` completion event. For example, if the `runOutputUri` is
+`s3://quilt-example/omics-quilt/3395667`, the package will be created in that
+same bucket with the name `omics-quilt/3395667`.
+
+### Workflow Run RO-Crate
+
+When enabled, this will create a package when ingesting any folder containing an
+When enabled, this will create a package when indexing any folder containing an
+`ro-crate-manifest.json`.  Indexing happens when the bucket is added to the stack,
+or when a folder is written to a bucket already in the stack.
+
+[RO-Crate](https://www.researchobject.org/ro-crate/) is a metadata standard for
+describing research data.  The Workflow Run working group adds three additional
+profiles, which are supported in the latest versions of
+[nf-prov](https://github.com/nextflow-io/nf-prov). You will need to explicitly
+configure `nf-prov` to use wrroc, by using a `nextflow.config` file [like
+this](https://github.com/famosab/wrrocmetatest):
+
+```groovy
+plugins {
+ id 'nf-prov@1.4.0'
+}
+
+prov {
+ enabled = true
+ formats {
+   wrroc {
+     file = "${params.outdir}/ro-crate-metadata.json"
+     overwrite = true
+     agent {
+       name = "John Doe"
+       orcid = "https://orcid.org/0000-0000-0000-0000"
+     }
+      license = "https://spdx.org/licenses/MIT"
+      profile = "provenance_run_crate"
+   }
+ }
+}
+```
+
+Note that Research Objects identify people using an ORCID iD, which anyone can
+get for free at [the ORDiD website](https://orcid.org/).
+
+The package will be created in the same bucket as the `outdir`, with the package
+name inferred from the folder key. For example, if the key is `my/s3/folder`,
+the package name will be `my_s3/folder`.
+
+## SQS Message Processing
+
+The primary interface to the Packaging Engine is through the
+SQS queue in the same account and region as your Quilt stack,
+listed in `PackagerQueue` under the Outputs. The queue URL will look something like:
+
+```text
+https://sqs.us-east-1.amazonaws.com/XXXXXXXXXXXX/PackagerQueue-XXXXXXXXXXXX
+```
+
+The body of the message is the stringified JSON of a package description.
+There is only one required parameter:
+
+```json
+{
+  "source_prefix": "s3://data_bucket/source/folder/metadata.json"
+}
+```
+
+This will create a package in the same bucket as the source folder, with the
+package name being inferred from the source URI.
+
+Optionally, you can control the package name, metadata, and other settings by
+explicitly specifying any of the following fields:
+
+```jsonc
+{
+  "source_prefix": "s3://data_bucket/source/folder/",  // If URI ends with slash, treats entire path as folder. Without slash, treats last segment as file and packages parent folder
+  "registry": "package_bucket",  // may be the same as `data_bucket`
+  "package_name": "prefix/suffix",
+  "metadata": { "key": "value" },  // object
+  "metadata_uri": "metadata.json", // alternative to `metadata`, relative or absolute
```

Comment:
@sir-sigurd What happens if they specify both types of metadata? Do we merge, or pick one?

Reply:
fail

Reply:
> The job will fail if you try to specify both `metadata` and `metadata_uri`.